app/code/Magento/AdminNotification/Block/Grid/Renderer/Actions.php

```diff
@@ -40,16 +45,16 @@ public function __construct(
      * @param   \Magento\Framework\DataObject $row
      * @return  string
      */
-    public function render(\Magento\Framework\DataObject $row)
+    public function render(DataObject $row) : string
     {
-        $readDetailsHtml = $row->getUrl() ? '<a class="action-details" target="_blank" href="' .
-            $this->escapeUrl($row->getUrl())
+        $readDetailsHtml = $row->getData('url') ? '<a class="action-details" target="_blank" href="' .
```

Comment:
Why `getData('url')` method call better instead of `getUrl`? I don't see benefits in such usage.

Reply:
To declare the key as a string reduces magic getters and makes the code more explicit.

Reply:
Methods like 'getUrl' can be defined in the class with specific logic instead of magic method call and by current changes, you may break functionality. In this case, all plugins defined for such methods also will not be called.
It would be better to revert such changes. 
Thank you.